### PR TITLE
Update main.c

### PR DESCRIPTION
**Pull request by dpjoshi**
I have added while loop to the main function. Please review.

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -3,5 +3,12 @@
 void main()
 {
  //Write your application code here.
+ 
+ //Add looping funcions here.
+ while(1)
+ {
+
```
Comment (frankc-cui-wr-fw) [inline main.c:11]: Must have ';' in this loop. This is just a joke to test the review feature :) .

**Review by frankc-cui-wr-fw — CHANGES_REQUESTED** [review]
I have added a comments, please modify the code

File: main.c (hunk 1 above)
Comment (dpjoshi) [reply main.c:11]: Hey! Thanks for pointing out that !! :)

**Review by frankc-cui-wr-fw — APPROVED** [review]
Code is great! I'll merge it to master, OK?

**Comment by frankc-cui-wr-fw** [comment]
Merge DJ's implementation of whileloop to master

**Pull request merged**